.repomixignore

```diff
@@ -0,0 +1,21 @@
+# Jest snapshots
+*.snap
+
+# Storybook stories
+*.stories.*
+
+# Mock data
+*.mock.*
+src/products/**/mock.js*
+
```

Comment:
Should `src/mocks/*` also get added here?

Reply:
Everything in there would get covered by `*.mock.*`, but no harm in including it. 👍